Server/src/server/dataCenter/DataCenter.java

```diff
@@ -48,6 +50,10 @@ public void run() {
         if (dataBase.isEmpty()) {
             GameServer.serverPrint("Reading Cards...");
             readAllCards();
+            boolean readCustomCards = Boolean.parseBoolean( Config.getInstance().getProperty("HOST_SERVER") );
```

Comment:
I don't think this should be the check. Shouldn't it always try to read all custom cards? Doesn't matter if the config is set up right for a client when the server is started... if the server is started it should try to load them.